test/tbb/test_eh_algorithms.cpp

```diff
@@ -1625,6 +1625,7 @@ void TestCancelation1_pipeline () {
     g_ThrowException = false;
     // Threshold should leave more then max_threads tasks to test the cancellation. Set the threshold to iter_range_size()/4 since iter_range_size >= max_threads*2
     intptr_t threshold = get_iter_range_size() / 4;
+    REQUIRE_MESSAGE(get_iter_range_size() - threshold > g_NumThreads, "Threshold should leave more then max_threads tasks to test the cancellation.");
```

Comment:
```suggestion
    // Threshold should leave more than max_threads tasks to test the cancellation. Set the threshold to iter_range_size()/4 since iter_range_size >= max_threads*2
    intptr_t threshold = get_iter_range_size() / 4;
    REQUIRE_MESSAGE(get_iter_range_size() - threshold > g_NumThreads, "Threshold should leave more than max_threads tasks to test the cancellation.");
```